tests/platform_tests/api/test_sfp.py

```diff
@@ -122,7 +122,8 @@ class TestSfpApi(PlatformApiTestBase):
                                                 'media_interface_technology',
                                                 'media_interface_code',
                                                 'host_electrical_interface',
-                                                'media_lane_assignment_option']
+                                                'media_lane_assignment_option',
```

Comment:
@mihirpat1 can we rename `EXPECTED_XCVR_NEW_QSFP_DD_OSFP_INFO_KEYS` -> `EXPECTED_XCVR_NEW_CMIS_INFO_KEYS` ?

Reply:
@prgeor Should I also change `EXPECTED_XCVR_NEW_QSFP_DD_OSFP_FIRMWARE_INFO_KEYS` to `EXPECTED_XCVR_NEW_CMIS_FIRMWARE_INFO_KEYS` ?